include/jsonbuilder/JsonBuilder.h

```diff
@@ -1254,62 +1263,98 @@ class JsonBuilder
     }
 
     /*
-    Creates a new value. Inserts it as the first (if front is true) or last
+    Creates a new value with the given name and data.
+    Inserts the value as the first (if front is true) or last
     (if front is false) child of itParent.
+
+    Name must be a string-view-like thing (pointer to nul-terminated
```

Comment:
>nul [](http://example.com/codeflow?start=54&length=3)

nit: null ?

Reply:
I generally use `NUL` to mean a character, and `NULL` to mean a pointer.